microsoft-edge/devtools-guide-chromium/index.md

```diff
@@ -120,10 +120,10 @@ DevTools grew over a period of several years.  You may assume that DevTools are
        Screenshot of the DevTools display of this article in an emulated mobile phone  
     :::image-end:::  
     
-*   The Tools tab group is a group of tabs that represent different tools that are used in different scenarios.  You may customize each of the tools and each tool may change based on the context.  To open a dropdown menu of more tools, choose the **More tabs** \(`>>`\) button.  Each of the tools is introduced later in the following section.  
-*   Next to the Tools tab group are optional error and issues shortcuts.  The shortcuts display when JavaScript errors or issues occur on the current webpage.  The **Open Console to view # errors, # warnings** \(**JavaScript Errors**\) button displays a red circle with an `X` followed by the number of JavaScript errors.  To open the [Console][DevtoolsGuideConsoleIndex] and learn about the error, choose the **JavaScript Errors** button.  The **Open Issues to view # issues** \(**Issues**\) button is a blue message icon followed by the number of issues.  To open the [Issues][DevtoolsGuideIssuesIndex] tool, choose **Issues** button.  
-*   The **Settings** button displays a gear icon.  To open DevTools **Settings** webpage, choose the **Settings** button.  The **Settings** webpage displays a menu to change **Preferences**, turn on **Experiments**, and much more.  
-*   The **Send Feedback** button displays torso with a chat bubble next to it.  To open the **Send Feedback** dialog, choose the **Send Feedback** button.  The **Send Feedback** dialog allows you to enter information to describe what happened and automatically includes a screenshot.  Use it to connect with the DevTools team to report problems, issues, or suggest ideas.  
+*   The Tools tab group is a group of tabs that represent different tools that are used in different scenarios.  You may customize each of the tools and each tool may change based on the context.  To open a dropdown menu of more tools, select the **More tabs** \(`>>`\) button.  Each of the tools is introduced later in the following section.  
```

Comment:
comment: These are good corrections to conform to Microsoft Writing Style Guide for readability.  Even if main purpose of PR is images not text.

Reply:
```suggestion
*   The Tools tab group is a group of tabs that represent different tools that are used in different scenarios.  You can customize each of the tools, and each tool may change based on the context.  To show a tool that is on a tab that's hidden, select the **More tabs** \(`>>`\) button.   To add a tool to the main toolbar or to the drawer, select the **More tools** (`+`) button.  Each of the tools is introduced later in the following section.  
```
Not sure where + and >> are covered.  Easier to do edit pass locally in VS Code.